.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
You can get rid of this. It does not belong in the repo.

Reply:
I'll add that to .gitignore then